.github/PULL_REQUEST_TEMPLATE/pr_ci.md

```
@@ -0,0 +1,36 @@
+name: "Build/CI PR"
+about: "Pipelines, linting, packaging, or release automation"
+title: "build(ci): <short summary>"
+labels: ["status:needs-review", "area:ci"]
+---
+<!-- Note: YAML front matter is parsed by our labeler workflow; GitHub itself does not apply labels from front matter. -->
+
+<!-- Suggested additional labels (pick as relevant):
+- compat:wordpress (if build targets)
+-->
+
+## Build/CI change
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)